pages/docs/tutorials/message-validation.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Message validation in runtime
+description: In this tutorial, you'll learn how to validate AsyncAPI messages (events).
+
+weight: 130

```

Comment:
@alequetzalli Kindly look into the weight as this weight may be changing the order of the pages inside the docs. You need to look inside the Tree presented on the Sidebar of Docs pages carefully.

![image](https://user-images.githubusercontent.com/76521428/208171271-e0b50429-b784-4d7f-bd94-9e3e789f0593.png)

I don't think you prefer to have this tutorial in between the Streetlights tutorial - Interactive and the normal Streetlights tutorial.

Reply:
@akshatnema I know you're not up-to-date on what's going on in the Docs roadmap, but your comment doesn't apply because both interactive and static streetlights tutorials will be removed when these are merged.

Thank you for reaching out and asking the question though! Always good to ask and understand what's going on right now